packages/overlay/src/integrations/sentry/data/sentryDataCache.ts

```diff
@@ -46,9 +47,14 @@ class SentryDataCache {
 
   setSidecarUrl(url: string) {
     const { origin } = new URL(url);
+    this.sidecarUrl = origin;
```

Comment:
If we go back to the `url.origin` we are not solving the problem reported in #524 as they want to be able to use the sidecar under a main root URL. That means we should let go of using `.origin` and slice the path up and just strip the last `/stream` to get the "root" sidecar URL. Makes sense?